resources/building-mobile-extensions/interacting-with-rules.md

```diff
@@ -8,9 +8,9 @@ Your extension should publish the rules support that it provides to Experience P
 
 ### Publishing supported data elements
 
-Data elements can be supported by publishing the shared state keys that you want to use in a rule. For more information, see [Data Element Types](https://developer.adobelaunch.com/extensions/reference/data-element-types/).
+Data elements can be supported by publishing the shared state keys that you want to use in a rule. For more information, see the Launch documentation on [data element types](https://developer.adobelaunch.com/extensions/reference/data-element-types/).
```

Comment:
this link is not found, can you find an alternative?
same for the other links on this page